src/inmanta/data/__init__.py

```diff
@@ -5859,24 +5859,22 @@ def to_dao(self) -> m.User:
         return m.User(username=self.username, auth_method=self.auth_method)
 
 
-class UnmanagedResource(BaseDocument):
+class DiscoveredResource(BaseDocument):
     """
     :param environment: the environment of the resource
-    :param unmanaged_resource_id: The id of the resource
-    :param values: The values associated with the unmanaged_resource
+    :param discovered_resource_id: The id of the resource
+    :param values: The values associated with the discovered_resource
     """
 
     environment: uuid.UUID
-    unmanaged_resource_id: m.ResourceIdStr
+    discovered_time: datetime.datetime
```

Comment:
I don't think the name `discovered_time` very clear. The `_time` part makes me think it expresses a certain duration, while the type indicates it's a timestamp. I think that `discovered_at` would be better for example..
```suggestion
    discovered_at: datetime.datetime
```